first_module/second.java

```diff
@@ -0,0 +1,9 @@
+public class second {
```

Comment:
This class duplicates the class below an should be removed as it is not in src folder